index.html

```diff
@@ -1086,9 +1086,25 @@
 
 ## Principles for Vulnerable People
 
-<aside class="issue">
-  This section has not been written.
-</aside>
+Some people, or groups of people, are at greater risk from the asymmetries of power created by the
```

Comment:
Do we need both "people" and "groups of people"? Can a group be at risk without its individual members being at risk?

In the "collective issues" section, it makes sense to talk about groups as separate from individuals, but I think the reasons there don't apply here.

Reply:
I would rather keep it as this can apply for instance to cohorts or to associations (say, data collected about unions or shelters).